### DISCUSSION TITLE
Issues applying for student benefits

### DISCUSSION THREAD

**Discussion by technobird22** [community/community#136400] [GitHub Education] [answered]
### Hi everyone,

Hello,

Recently, my student pack expired so I wanted to try renewing it with my new student ID. However, whenever I try accessing the application for benefits, I get redirected to the [Github Pricing](https://github.com/pricing) page.

Currently, I am unable to use any of the benefits from the student developer pack, and it would be very useful to have some of these for some projects I'm working on. I'd really appreciate if it could be resolved soon.

Thanks,
Albert

**Comment by pkprajapati7402** [comment]
That's in the Rules of GitHub Pricing policy. Like you already have been benefitted with the student developer pack. And again you want to get the student developer pack. So let me clear you, Student developer pack can be accesed only once on an ID. you don't have option to renew it.
and it's obvious that you have used the products of student developer pack, and you like the tools. Also the tools that was provided to you was for trial purposes, so that you will use those tools and softwares for further development and you will pay as per Business Rules and Marketing Strategy.

But here's a hack. Go and take ID of your sibling, or your non tech friend, and make another Account and get those perks on the another account. If you are working on projects on your previous ID, just make it admin id in Your New Benefitted ID. And you are good to go.

Enjoy your Student Developer Pack.

**Comment by davevad93** [comment, marked as answer]
Hi @technobird22 ,

You aren't the only one experiencing this problem, as you can see in these discussions, #131668 , #135388 , #135546 the page to renew your benefits is [this](https://education.github.com/discount_requests/application), although If the link I have provided still redirects you to the pricing page, try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your problem.

**Reply by technobird22** [reply]
Hey Davide, thanks for your answer! Unfortunately, the renewal link still does not work, so I'll try opening a ticket.
Thanks again!

**Reply by technobird22** [reply]
Edit: Thanks for the suggestion! I just rechecked and it seems that they have replied and resolved the issue!
Hope you have a wonderful day

**Reply by davevad93** [reply]
Glad the issue has been resolved @technobird22 , have a great day too! 👍
